backend/hitas/views/condition_of_sale.py

```diff
@@ -140,11 +141,9 @@ def get_household(owner_uuids: list[uuid.UUID]) -> list[Owner]:
                             .order_by("purchase_date")
                             .values_list("id", flat=True)[:1]
                         )
-                    ).only("id", "purchase_date"),
+                    ),
                 ),
-            )
-            .only("id")
-            .filter(uuid__in=owner_uuids)
+            ).filter(uuid__in=owner_uuids)
```

Comment:
Unfortunately, I don't think we can filter out the owners with `create_conditions_of_sale=False` here, since we have a check after this whether we got all the owners we were supposed to. There is no way to know whether the owner was not fetched because the owner id was wrong, or if it should not have conditions of sale created for it.